qsirecon/tests/data/pyafq_recon_external_trk_optional_outputs.txt

```diff
@@ -2,4 +2,4 @@ derivatives/qsirecon-PYAFQ/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-T1w_dwimap/b
 derivatives/qsirecon-PYAFQ/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-T1w_dwimap/bundles/sub-ABCD_acq-10per000_coordsys-RASMM_trkmethod-probCSD_recogmethod-AFQ_desc-LeftSuperiorLongitudinal_tractography.trk
 derivatives/qsirecon-PYAFQ/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-T1w_dwimap/viz_core_bundles/sub-ABCD_acq-10per000_coordsys-RASMM_trkmethod-probCSD_recogmethod-AFQ_desc-LeftArcuateviz_dwi.html
 derivatives/qsirecon-PYAFQ/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-T1w_dwimap/viz_core_bundles/sub-ABCD_acq-10per000_coordsys-RASMM_trkmethod-probCSD_recogmethod-AFQ_desc-LeftSuperiorLongitudinalviz_dwi.html
-derivatives/qsirecon-PYAFQ/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-T1w_dwimap/viz_core_bundles/sub-ABCD_acq-10per000_space-T1w_dwi_space-RASMM_model-probCSD_algo-AFQ_desc-ARCLviz_dwi.html
+derivatives/qsirecon-PYAFQ/sub-ABCD/dwi/sub-ABCD_acq-10per000_space-T1w_dwimap/viz_core_bundles/sub-ABCD_acq-10per000_space-T1w_dwi_space-RASMM_model-probcsd_algo-AFQ_desc-ARCLviz_dwi.html
```

Comment:
I was wondering how much control over the pyafq output naming we have. Are some of these upper and some lower CSDs?